Application/Frontend/components/Messaging/AblyWrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import dynamic from 'next/dynamic';
```

Comment:
Indeed, that was there because the Ably `useContext` hook was causing an extremely annoying error where the provider wrapper could not be loaded on the server-side despite its actual use in only client-side components, hence attempting to load it in a client-side context such that Next.js is aware that it is not being rendered as a server-side wrapper. This was scrapped and replaced with a sipmle "use client" addition at the top of the `layout.tsx` file (which is not even supposed necessary, but this seems like a new bug with the Ably provider).